### ISSUE TITLE
Data originator wants to assign *parameter names* as desired

### ISSUE THREAD

**Issue by rico-hengst** [halo-db/storymap#33]
Data originator wants to upload his data. But he wants to assign *parameter names* as desired. He do not want use any naming convention of the scientific community, eg: [CF metadata convention](https://cfconventions.org/Data/cf-standard-names/77/build/cf-standard-name-table.html)

**Comment by sabarth** [comment]
...in times of coordinated and "FAIR" datamanagement, one should try to prevent individual naming to ease correct usage of data and comparability with other data sets.

**Comment by joerg-halo** [comment]
This would prevent automated searches and running other scripts, right?

**Comment by rico-hengst** [comment]
> This would prevent automated searches and running other scripts, right?

Yes. The use cas was intentionally created in contrast to the wishes of other users who would like to search for data according to defined criteria https://github.com/halo-db/storymap/issues/32 . Hence, the protocol of the discussion of this use case is archived there.
